test/controllers/api/v2/compute_resources_controller_test.rb

```diff
@@ -498,4 +498,22 @@ def teardown
       assert_response :unprocessable_entity, "Can create libvirt compute resource with invalid name"
     end
   end
+
+  context 'ovirt' do
+    let(:simple_json) do
+      {
+        name: "oVirtTest-#{Foreman.uuid}",
+        provider: "Ovirt",
+        datacenter: "a221bee8-d53b-438e-8c43-fc9dc2b770b3",
+        url: "https://localhost/ovirt-engine/api",
+        user: "admin@internal",
+        password: "changeme"
+      }
+    end
+
+    test 'Should create oVirt resource' do
+      post :create, params: { compute_resource: simple_json }
+      assert_response :success, 'Unable to add oVirt compute resource'
```

Comment:
'Unable to add oVirt compute resource' looks like an error message, is that what we actually return on success? is so please open a bug about it.

Reply:
my bad, there is no message